crates/test_ui/src/services/requests.rs

```diff
@@ -0,0 +1,130 @@
+use dotenv_codegen::dotenv;
+use gloo::storage::{LocalStorage, Storage};
+use once_cell::sync::Lazy;
+use parking_lot::RwLock;
+use serde::{de::DeserializeOwned, Serialize};
+
+use crate::{error::Error, types::ErrorInfo};
+
+const API_ROOT: &str = dotenv!("API_ROOT");
+const TOKEN_KEY: &str = "yew.token";
+
+////////////////////////////////////
+// Token
+////////////////////////////////////
+
+/// Jwt token read from local storage.
+pub static TOKEN: Lazy<RwLock<Option<String>>> = Lazy::new(|| {
+    if let Ok(token) = LocalStorage::get(TOKEN_KEY) {
+        RwLock::new(Some(token))
+    } else {
+        RwLock::new(None)
+    }
+});
+
+/// Get jwt token from lazy static.
+pub fn get_token() -> Option<String> {
+    let token_lock = TOKEN.read();
+    token_lock.clone()
+}
+
+/// Set jwt token to local storage.
```

Comment:
for information, server uses https://www.biscuitsec.org/docs/Usage/rust/ ; not jwt